src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Resolve.java

```diff
@@ -1335,9 +1335,9 @@ public void visitParens(JCParens tree) {
                 @Override
                 public void visitLambda(JCLambda tree) {
                     if (sRet.hasTag(VOID)) {
-                        result &= true;
+                        result = true;
```

Comment:
Same as above, the change is changing the semantics:
original code keeps the value of `result` while the new one sets it to `true`.